website

```diff
@@ -96,19 +96,21 @@
     <h1>TBD</h1>
     <div id="slider">
         <figure>
-            <img src="">
-            <img src="">
-            <img src="">
-            <img src="">
+            <img src="C:\Users\andre\OneDrive\Desktop\_cgi-bin_mmwebwx-bin_webwxgetmsgimg__&MsgID=70839570771797268&skey=@crypt_d1e2b980_cf756669f8eac8d7337b5e2646fed22b&mmweb_appid=wx_webfilehelper.jpg">
```

Comment:
These links won't work as they are only available on your computer; they're not online. You should add the images to this repo under a directory like `./media/` and then you can reference them here as `./media/webfilehelper.jpg` or similar. So steps would be:

1. Create a directory called `media` within the repo
2. Add any images you need for the webpage in there
3. Change all the links to reference them there
4. `git add media`
5. `git add website`
6. `git commit -m "updating images to github"
7. `git push`

Let me know if you have any questions.